packages/cli/internal/pkg/cli/zipfile/compress_to_tmp.go

```diff
@@ -24,16 +27,33 @@ func CompressToTmp(srcPath string) (string, error) {
 	return packFile.Name(), nil
 }
 
+func expandHomeDir(rootPath string) (string, error) {
```

Comment:
This method should be moved to another package (can be useful outside zip package).
Options:
1) move it to `config` package
2) crate a new package `osutils` and move both DetermineHomeDir and ExpandHomeDir to this new package

Reply:
I created an `osutils` package and moved the relevant functions there.